spec/models/event_spec.rb

```diff
@@ -72,4 +72,30 @@
     expect(Event.upcoming.include?(@event)).to be(true)
   end
 
+  describe '#next_event' do
+    let(:event_001) { create :event, start_time: "2017-11-02 03:36:00", end_time: "2017-11-04 03:36:00" }
+    let(:event_002) { create :event, start_time: "2017-11-03 03:36:00", end_time: "2017-11-06 03:36:00" }
+    let(:result) { event_001.next_event }
+    before do 
+      event_002
+    end
+
+    it 'should return event_002' do
+      expect(result).to eq event_002
```

Comment:
These tests would be much more valuable if you added an `event_000` and an `event_003`. Think if someone changed the code to get the last event, not the next. These tests would still pass.
Also, if you put this in a context, your `let` statements could happen once, eliminating the Code Climate complaint.

Reply:
Yeah, good.